en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -116,6 +113,10 @@ This section explains how to take a regular DataObject and add versioning to it.
 
 ### Applying the `Versioned` extension to your `DataObject`
 
+> [!WARNING]
+> Versioning only works if you are adding the extension to the base class. That is, the first subclass
+> of `DataObject`. Adding this extension to children of the base class will have unpredictable behaviour.
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.